querybook/webapp/components/DataDoc/DataDoc.tsx

```diff
@@ -427,9 +428,18 @@ class DataDocComponent extends React.PureComponent<IProps, IState> {
                     }
                 };
                 if (shouldConfirm) {
+                    let cellName = '';
```

Comment:
this part is same as 
```ts
cellName = ` ${cell.meta.title ?? ''}`
```

Reply:
Structured this part in this way since I was getting the below linting error. 
```
Property 'title' does not exist on type 'IDataQueryCellMeta | IDataCellMetaBase | IDataChartCellMeta'.
  Property 'title' does not exist on type 'IDataCellMetaBase'.
```